src/ckeditor/ckeditor.component.spec.ts

```diff
@@ -239,6 +239,23 @@ describe( 'CKEditorComponent', () => {
 				} );
 			} );
 
+			describe( 'on destroy', () => {
+				it ( 'should not have call runOutsideAngular when destroy before DOM loaded', done => {
+					spyOn( fixture.ngZone, 'runOutsideAngular' );
+
+					fixture.detectChanges();
+
+					waitUntil( () => {
+						fixture.destroy();
+						return true;
+					}, 200 )
```

Comment:
Is there any particular reason for 200ms? Could it be less (or even \`0\`)?

Reply:
hello it could be 0 but I wanted to test with a bit of latency